### PR TITLE
Queries contain locations

### PR DESCRIPTION
**Pull request by AttackOnMorty**
This PR contains those queries:

- [Who is on client work?](https://github.com/SSWConsulting/SSW.SophieBot/issues/18)
- [Who is on client work next Friday?](https://github.com/SSWConsulting/SSW.SophieBot/issues/21)
- [Who is free next Friday?](https://github.com/SSWConsulting/SSW.SophieBot/issues/19)
- [Who is working for SSW Today?](https://github.com/SSWConsulting/SSW.SophieBot/issues/22)

### CODE REVIEW

**Comment by jimmidier** [comment]
> [Who is on client work?](https://github.com/SSWConsulting/SSW.SophieBot/issues/18)
- [ ] Change column header style for dark mode. image
- [ ] use `messageBack` in action data

**Review by jimmidier — CHANGES_REQUESTED** [review]
- [ ] Change column header style for dark mode. image
- [ ] use `messageBack` in action data

File: src/SSW.SophieBot/SSWSophieBot/dialogs/GetPeopleOnClientWorkDialog/language-generation/en-us/GetPeopleOnClientWorkDialog.en-us.lg [hunk 1]
```diff
@@ -27,54 +28,80 @@
...
+        "internal": "${dialog.internalEntity}",
+        "location": "${location}",
+        "datetime": [
+            {
+            "timex": [
+                "${dialog.fromDate}"
+            ],
+            "type": "date"
+            }
+        ]
```
Comment (jimmidier) [inline src/SSW.SophieBot/SSWSophieBot/dialogs/GetPeopleOnClientWorkDialog/language-generation/en-us/GetPeopleOnClientWorkDialog.en-us.lg:100]:
```suggestion
        ],
        "msteams": {
            "type": "messageBack",
	    "displayText": "${location} [${number}]"
        }

```

File: src/SSW.SophieBot/SSWSophieBot/dialogs/GetPeopleOnClientWorkDialog/language-generation/en-us/GetPeopleOnClientWorkDialog.en-us.lg [hunk 2]
```diff
@@ -27,54 +28,80 @@
...
+                        "internal": "${dialog.internalEntity}",
+                        "location": "${location}",
+                        "datetime": [
+                            {
+                            "timex": [
+                                "${dialog.fromDate}"
+                            ],
+                            "type": "date"
+                            }
+                        ]
```
Comment (jimmidier) [inline src/SSW.SophieBot/SSWSophieBot/dialogs/GetPeopleOnClientWorkDialog/language-generation/en-us/GetPeopleOnClientWorkDialog.en-us.lg:44]:
```suggestion
                        ],
                        "msteams": {
                            "type": "messageBack",
			    "displayText": "Show All [${count(employees)}]"
                        }

```

File: src/SSW.SophieBot/SSWSophieBot/dialogs/GetPeopleOnClientWorkDialog/language-generation/en-us/GetPeopleOnClientWorkDialog.en-us.lg (hunk 1 above)
Comment (AttackOnMorty) [reply src/SSW.SophieBot/SSWSophieBot/dialogs/GetPeopleOnClientWorkDialog/language-generation/en-us/GetPeopleOnClientWorkDialog.en-us.lg]: The number after the location is not necessary. Will remove it.

**Review by jimmidier — APPROVED** [review]

**Pull request merged**